### DISCUSSION TITLE
How to disable "Select All" option from the top-bar checkbox?

### DISCUSSION THREAD

**Discussion by VitaliyMinenko** [laravel/nova-issues#5783] [Q&A]
Is it possible to remove the "Select All" option from the top bar? I still want the "Select this page" option.

![image](https://github.com/laravel/nova-issues/assets/11076616/ec3df659-17ea-4560-b730-41fcc81634e8)

**Comment by treed-eaglemetal** [comment]
I don't see a way to remove the "Select All" option from the top bar that's easy.

You could swap out the `SelectAllDropdown` Vue component with your own, but that's risky business for some folk.
